controllers/usersController.js

```diff
@@ -1,54 +1,39 @@
 const Database  = require('../model/database');
-const User   = require('../model/user');
+const Account   = require('../model/account');
+const User      = require('../model/user');
 const UsersController = module.exports;
-const Authentication = require('../model/authentication');
 
 const COLLECTION_NAME = 'Users';
 
-const ALLOWED_ORIGIN = 'http://localhost:3000'; 
-
-UsersController.preflight = function(req, res) {
-
-    res.setHeader('Access-Control-Allow-Origin', ALLOWED_ORIGIN); 
-    res.setHeader('Access-Control-Allow-Methods', 'POST, GET, OPTIONS'); 
-    res.setHeader('Access-Control-Allow-Headers', 'authorization, content-type');  
-    res.sendStatus(200); 
-
-}
-
-
```

Comment:
If you allow each controller to have its own preflight handler, then you can define which methods are allowed on a per-route basis (Principle of Least Privilege). However since it's not a public API and we're not concerned about malicious attempts, there's nothing to change here.

Reply:
I'm pretty sure the CORS isn't even a concern anymore actually, since we're running them on the same domain. I think @MrNickCasey is making the calls in no-cors mode.

Reply:
Okay, but we'll need it for the dashboard and mobile apps, right?

Reply:
That's a good point, yeah. I'd prefer having just a single preflight instead of having one per controller, since I don't think there's anything that'll change much per controller (I think they'll all be used by the apps, and they'll all provide more or less the same methods)